### PR TITLE
[docs] updates to diagnostics report page

### PR DESCRIPTION
**Pull request by aishwarya24**
https://deploy-preview-23162--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-voyager/diagnostics-report/

### CODE REVIEW

File: docs/content/preview/yugabyte-voyager/diagnostics-report.md [hunk 1]
```diff
@@ -19,68 +19,180 @@ None of your data or personally identifiable information is collected.
...
+  - Total cores on the target YugabyteDB cluster
+- Time taken in the migration phase run till the point payload is collected
+- Status of the phase while payload is sent (for example, In-progress, Complete, Exit, and so on.)
+- Client machine's IP on which voyager is running
+
+Data collected in `Phase payload` as per migration phases are described in the following table:
+
+| Migration phase | Data collected |
+| :---- | :------------- |
+| [assess-migration](../reference/assess-migration/) | <li> Unsupported features (reported in the assessment report)</li> <li>Unsupported datatypes (reported in the assessment report)</li> <li>Error message while running the assessment if any </li> <li>Table sizing statistics (table name, reads/writes per second and size in bytes)</li> <li>Index sizing statistics (index name, reads/writes per second and size in bytes)</li> <li>Schema summary (reported in the assessment report)</li> <li>Source connectivity (whether assessment is run with source connectivity) </li> <li>Command line arguments changed during this phase<sup>1</sup> </li>|
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/diagnostics-report.md:46]: think you need to wrap all the lists in <ul> to get the indentation right

File: docs/content/preview/yugabyte-voyager/diagnostics-report.md [hunk 2]
```diff
@@ -19,68 +19,180 @@ None of your data or personally identifiable information is collected.
...
+| [import-data](../reference/data-migration/import-data/#import-data) <br/> [import-data-to-target](../reference/data-migration/import-data/#import-data) <br/> [import&#8209;data&#8209;to&#8209;source&#8209;replica](../reference/data-migration/import-data/#import-data-to-source-replica) <br/>[import-data-to-source](../reference/data-migration/import-data/#import-data-to-source) | <li>Parallel jobs used </li><li>Total rows (total number of rows of all tables imported) </li>Largest table rows(total number of rows of all tables imported)</li> <li>Phase (Snapshot or Streaming or Cutover)</li><li>Total imported events (in case of live migration these are the total imported events)</li><li>Events import rate  (in case of live migration this the rate of events imported for the last 3 minutes)</li> <li>Live migration workflow type (Normal, with fall-forward or fall-back)</li><li>Command line arguments changed during this phase<sup>1</sup></li> |
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/diagnostics-report.md:51]:
```suggestion
| [import-data](../reference/data-migration/import-data/#import-data) <br/> [import-data-to-target](../reference/data-migration/import-data/#import-data) <br/> [import&#8209;data&#8209;to&#8209;source&#8209;replica](../reference/data-migration/import-data/#import-data-to-source-replica) <br/>[import-data-to-source](../reference/data-migration/import-data/#import-data-to-source) | <li>Parallel jobs used </li><li>Total rows (total number of rows of all tables imported) </li><li>Largest table rows (total number of rows of all tables imported)</li> <li>Phase (Snapshot, Streaming, or Cutover)</li><li>Total imported events (for live migration these are the total imported events)</li><li>Events import rate (for live migration this the rate of events imported for the last 3 minutes)</li> <li>Live migration workflow type (Normal, with fall-forward or fall-back)</li><li>Command line arguments changed during this phase<sup>1</sup></li> |
```

File: docs/content/preview/yugabyte-voyager/diagnostics-report.md [hunk 3]
```diff
@@ -19,68 +19,180 @@ None of your data or personally identifiable information is collected.
...
+| [import-data-file](../reference/bulk-data-load/import-data-file/) (bulk data load) | <li>Parallel jobs used </li><li> Total size (total size of all tables bulk imported) </li><li>Largest table size (largest size of a tables imported) </li> <li>File storage type (Local, AWS S3, GCS, or Azure blob storage) </li><li>Data file format parameters (for example, null-string, format type, and so on.) |
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/diagnostics-report.md:52]:
```suggestion
| [import-data-file](../reference/bulk-data-load/import-data-file/) (bulk data load) | <li>Parallel jobs used </li><li> Total size (total size of all tables bulk imported) </li><li>Largest table size (largest size of tables imported) </li> <li>File storage type (Local, AWS S3, GCS, or Azure blob storage) </li><li>Data file format parameters (for example, null-string, format type, and so on) |
```

File: docs/content/preview/yugabyte-voyager/diagnostics-report.md [hunk 4]
```diff
@@ -19,68 +19,180 @@ None of your data or personally identifiable information is collected.
...
+
+<sup>1</sup> Note that no sensitive parameters like password related arguments are stored.
+
+The following are examples of sample payloads that are collected during some migration phases:
+
+- `assess-migration` command execution:
+
+  ```output
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/yugabyte-voyager/diagnostics-report.md:62]: json.. modified it.. hopefully looks better now.

File: docs/content/preview/yugabyte-voyager/diagnostics-report.md [hunk 5]
```diff
@@ -19,66 +19,301 @@ None of your data or personally identifiable information is collected.
...
-- TargetDB Version (Target YugabyteDB version)
-- Total Rows (total number of rows of all tables migrated)
-- Total Size (total number of size of all tables migrated)
-- Largest Table Rows (maximum number of rows migrated for a table)
-- Largest Table Size (maximum size of data migrated for a table)
-
-The following is an example of a payload that is collected:
-
-```output
+- Migration Phase name (for example, export-data, import-schema, and so on.)
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/diagnostics-report.md:22]:
```suggestion
- Migration phase name (for example, export-data, import-schema, and so on)
```

File: docs/content/preview/yugabyte-voyager/diagnostics-report.md [hunk 6]
```diff
@@ -19,66 +19,301 @@ None of your data or personally identifiable information is collected.
...
+- Migration Phase Start Time of a particular run
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/diagnostics-report.md:23]:
```suggestion
- Migration phase start time
```

**Review by ddhodge — APPROVED** [review]

**Pull request merged**